include/micm/solver/jit_rosenbrock.hpp

```diff
@@ -87,7 +87,7 @@ namespace micm
       MatrixPolicy<double> temp{};
       if (temp.GroupVectorSize() != parameters.number_of_grid_cells_)
       {
-        throw std::runtime_error("Number of grid cells for JitRosenbrockSolver must match template parameter.");
+        throw std::system_error(make_error_code(MicmJitErrc::InvalidMatrix), "");
```

Comment:
Might be worth having a message why it's invalid?

Reply:
done!